jpells/jblog
# add edit functionality to posts

# Added Edit functionality

Hey!I just added an edit feature that I think will make managing blog way easier. 

# Changes made
- Created an awesome `/edit/:filename` page where admins can tweak blog posts
- Built an `/update` endpoint that saves changes super smoothly
- Converted existing posts to Markdown for super-easy editing

## To Test
- Run `go test ./...`
- Try editing a post and see the magic happen!

Thoughts? Let me know what you think! 🤔
